server/core/service.cc

```diff
@@ -2425,3 +2427,8 @@ void Service::update_basic_parameter(const std::string& key, const std::string&
         retain_last_statements = std::stoi(value);
     }
 }
+
+uint64_t mxs_lru_clock()
```

Comment:
This could probably be renamed from `mxs_lru_clock` to `mxs_logical_clock_tick` to tell the reader that it modifies the value.